docs/root/configuration/cluster_manager/cluster_stats.rst

```diff
@@ -130,10 +130,14 @@ statistics will be rooted at *cluster.<name>.outlier_detection.* and contain the
   ejections_overflow, Counter, Number of ejections aborted due to the max ejection %
   ejections_enforced_consecutive_5xx, Counter, Number of enforced consecutive 5xx ejections
   ejections_detected_consecutive_5xx, Counter, Number of detected consecutive 5xx ejections (even if unenforced)
-  ejections_enforced_success_rate, Counter, Number of enforced success rate outlier ejections
-  ejections_detected_success_rate, Counter, Number of detected success rate outlier ejections (even if unenforced)
+  ejections_enforced_success_rate, Counter, Number of enforced success rate outlier ejections for external failures
```

Comment:
This is not always true if we are not splitting. Same elsewhere. Can you audit docs to make this more clear across all the docs in case there are other similar places?

Reply:
Corrected.